packages/blade/src/components/BaseHeaderFooter/BaseHeader.tsx

```diff
@@ -230,7 +234,12 @@ const _BaseHeader = ({
                 flexDirection="row"
               >
                 {title ? (
-                  <Text size="large" weight="semibold" color="surface.text.gray.normal">
+                  <Text
+                    size="large"
+                    marginTop={makeSize(size['1'])}
+                    weight="semibold"
+                    color="surface.text.gray.normal"
+                  >
```

Comment:
We are changing things on BaseHeader, make sure this doesn't not effect Modal, BottomSheet, ActionList in any negative way.

Reply:
nope. it is expected change for all of them. I checked on BaseHeader stories itself